internal/grouper/grouper_models.go

```diff
@@ -23,12 +28,14 @@ func ConvertVulnerabilityToIDAliases(c []osvschema.Vulnerability) []IDAliases {
 		// all related CVEs should be bundled together, as they are part of this DSA.
 		// TODO(gongh@): Revisit and provide a universal way to handle all Linux distro advisories.
 		if strings.Split(v.ID, "-")[0] == "DSA" {
+			idAliases.Aliases = append(idAliases.Aliases, v.Upstream...)
```

Comment:
We should be able to apply this to all vulnerabilities right? Not only for DSA and USN. This also means we should stop appending related ones to it. But given the fact that Ubuntu has updated everything from "related" into "upstream," maybe we should write a "todo" here to remind ourselves.